File: Java/FindDuplicatesInArray.java
// To find duplicates values in array

public class FindDuplicatesInArray {
    public static void main(String[] args) {
        int arr[]={1,2,3,2,4,1};
        System.out.println("Original Array: ");
        for(int i=0;i<arr.length;i++){
            System.out.print(arr[i]+" ");
        }
        System.out.println("\nRepeated values: ");
        for(int i=0;i<arr.length;i++){
        boolean flag=false;
            for(int j=i+1;j<arr.length;j++){
                if(arr[i]==arr[j]){
                    flag=true;
                }
                if(flag){
                    System.out.println(arr[i]);
                    flag=false;
                }
            }
        }
    }
}

File: python/FindDuplicatesInArray.py
# To find duplicates values in array
arr=[1,2,3,4,3,1]
flag=False
print("Original Array: ",arr)
print("Repeated values: ")
for i in range(0,len(arr)):
    for j in range(i+1,len(arr)):
        if(arr[i]==arr[j]):
            flag=True
        if(flag):
           print(arr[i])
           flag=False